src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -38,7 +38,8 @@
       (let [{:keys [name color balance watch-only?]} (rf/sub [:wallet/current-viewing-account])
             currency-symbol                          (rf/sub [:profile/currency-symbol])]
         [rn/view {:style {:flex 1}}
-         [account-switcher/view {:on-press #(rf/dispatch [:wallet/close-account-page])}]
+         [rn/pressable {:on-press #(rf/dispatch [:navigate-to :wallet-transaction-progress])}
```

Comment:
temporary navigation on account page above account overview. I prefer to remove this from pr but will leave while it is being reviewed if any devs want to check out the page quickly.

Reply:
Please add a note in the PR on where we need to tap (should tap on the empty area of the navigation header of the account screen). I was searching for a button or something on the page. 😄

Reply:
Sorry yeah it's pretty unclear 😆